.nvmrc

```diff
@@ -0,0 +1 @@
+12.16.3
```

Comment:
Ah, I actually just found this in `app/`. Let me know what you want me to do here, i.e. happy to update the instructions to `cd app`.

Reply:
actually I'll delete the one in app/ !

now there is the actions-server subdir (it's a node express server), we should probably use the same version of node across both :)